wagtail_wordpress_import/prefilters/transform_styles_defaults.py

```diff
@@ -43,40 +43,50 @@ def transform_style_center(soup, tag):
     """
     apply a new css class to any existing classes
     """
-    _class = tag.get("class", "") + " align-center"
-    tag.attrs["class"] = _class.strip()
+    if tag.attrs.get("class"):
+        tag.attrs["class"].append("align-center")
+    else:
+        tag.attrs["class"] = "align-center"
```

Comment:
**Question**: Is this parsed with BeautifulSoup4 as HTML? If so, these should be multivalued attributes, and should already return a string if you access `tag["class"]`. See https://www.crummy.com/software/BeautifulSoup/bs4/doc/#multi-valued-attributes.

In that case we can use:

```suggestion
    tag["class"].append("align-centre")
```

Reply:
Thanks Nick but because tag["class"] isn't always present if I change the above to your suggestion i see a `KeyError` if thats the case we need to add the attribute `class`

A soup object is passed in thats been parsed with `html.parser`